CONTRIBUTING.md

```diff
@@ -2,44 +2,58 @@
 
-All definitions are stored in the [definitions folder](11ty/definitions/). Create a file named `word.md` in it. `word` should be the name of the word you are defining. Not `word`. Except you are defining «word». In this case it should be `word`. For this tutorial I will take the word «Obsessive Compulsive Disorder», as it allows me to show some of the feature. That means I would create the file `obsessive-compulsive-disorder.md` (as it is a defined word, you can have a look at [the finished definition](11ty/definitions/obsessive-compulsive-disorder.md)).
+- Step 1: Create a file named `word.md` in in the [definitions folder](11ty/definitions/) where we store all definitions.
 
-All definitions consist of two parts: Front Matter and content. Front Matter is technical bubble for some structured data about your content. [You can find the complete documentation on our website](https://www.selfdefined.app/documentation/front-matter/). Here’s a quick run down:
+  `word` should be the name of the word you are defining. Not `word`. Except you are defining «word». In this case it should be `word`. 
 
-Every definition _needs_ to have a `title`. In my case «Obsessive Compulsive Disorder». And a slug. That’s the string we need to link somewhere. And it needs a `slug`. A slug is URL friendly, hyphenated version of the title. In my case: `obsessive-compulsive-disorder`. The third required item is the `defined` flag. This is either `true` or `false`. As soon as you set it to `true` your definition will be visible. Finally we need to what kind of word your word. A noun? Or adjective? That’s what `speech` is for. In my example it is a noun.
+  For this tutorial I will take the word «Obsessive Compulsive Disorder», as it allows me to show some of the features. That means I would create the file `obsessive-compulsive-disorder.md`. As it is already defined, you can have a look at the [finished definition](11ty/definitions/obsessive-compulsive-disorder.md).
 
-This leaves us with the following front matter:
+- Step 2: Populate the file you created.
 
-```yaml
----
-title: Obsessive Compulsive Disorder (OCD)
-slug: obsessive-compulsive-disorder
-defined: true
-speech: noun
----
+  All definitions consist of two parts: **front matter** and **content**. Front Matter is a technical bubble for some structured data about your content. On our website you can find the [complete documentation on Front Matter](https://www.selfdefined.app/documentation/front-matter/). Here’s a quick run down:
 
-```
+  Every definition _needs_ to have this meta information: `title`, `slug`, `defined`, and `speech`. Let's discuss what each of these mean:
+  
+  - `title` is easy. In my case it's «Obsessive Compulsive Disorder». 
+  
+  - `slug` is a string that we'll use to link to this word. It should be a URL-friendly, hyphenated version of the title. In my case: `obsessive-compulsive-disorder`. 
```

Comment:
@ovlb Should we here suggest that it should be lowercase?

In the future we'll need to consider abbreviations and parsing; should we suggest here that it's the "full title?"

Reply:
Full title and lower case make sense to be added here.

Reply:
@ovlb @tatianamac is it always full title though? In the example the title is _"Obsessive Compulsive Disorder (OCD)"_, but the slug is _obsessive-compulsive-disorder_ without the _ocd_ part.

Reply:
I've made these changes, but the question about full title still stands

Reply:
Good point. I lean towards omitting abbreviations in the slug, since they are additional information to the actual title of the word. But that’s personal preference, I guess.

Reply:
I think to be explicit we should say something like (feel free to edit it for clarity, ofc):

"Please include the full phrase but omit any abbreviations or initialisms that are included in the `title`. For example, we omitted OCD."